test/PublishCommand.js

```diff
@@ -1170,6 +1170,107 @@ describe("PublishCommand", () => {
   });
 
   /** =========================================================================
+   * NORMAL - EXACT CONFIG
```

Comment:
I appreciate test cases, although I think in this case we've already covered the ability for "durable" `getOptions()` config to bubble up through the various levels (`lerna.json` vs CLI flags, etc).

I won't say "no" to more tests, though. ^_^